pkg/target/regolib/src.rego

```diff
@@ -267,6 +269,16 @@ get_ns[out] {
   out := data["{{.DataRoot}}"].cluster["v1"]["Namespace"][input.review.namespace]
 }
 
+get_ns[out] {
```

Comment:
This should be a separate rule `get_ns_name` maybe?

`get_ns` returns the full namespace object.